build/configs/qemu/HowToRunNetworkStackOnQemu.md

```diff
@@ -0,0 +1,80 @@
+# Steps to verify network stack in QEMU is as follows
+1. cd os/tools
+2. ./configure.sh qemu/tc_16m
+3. cd ..
+4. make
+
+Connect WiFi AP to Linux machine(wired connection. dhcp should be enabled in the AP)
```

Comment:
1 . Please add (mark) '.' at end of every sentence.
2.  In this line, please add two spaces at end of line to change a line.